public/stylesheets/style.css

```
@@ -1,12 +1,45 @@
 body {
   padding: 50px;
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  background-color: #1F1F1F;
+  color: #ffffff;
 }
```

Tipp: avoid to declare tags, ids, and classes twice so that we do not lose track when the file becomes more extensive

```suggestion
body {
  padding: 50px;
  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
  background-color: rgba(2, 2, 2, 0.837);
  color: #ffffff;
}
```